bin/pyntlm_auth/config_loader.py

```diff
@@ -168,6 +168,16 @@ def config_load():
                 print(f"  NT key cache: expire value '{nt_key_cache_expire}' too large, set to maximum value: 864000")
                 nt_key_cache_expire = 864000
 
+            ad_old_password_allowed_period, error = get_int_value(ad_old_password_allowed_period)
+            if error is not None:
+                print(f"  NT Key cache: unable to parse 'ad_old_password_allowed_period', cache disabled.")
+                nt_key_cache_enabled = False
+                break
+            if ad_old_password_allowed_period < 0 or ad_old_password_allowed_period > 99999:
+                print(f"  NT Key cache: 'ad_old_password_allowed_period' ranges from 0..99999, cache disabled.")
+                nt_key_cache_enabled = False
+                break
```

Comment:
0 and 99999 are valid ad_old_password_allowed_peroid. 
if it's set to 0, no old password will be accepted, changing password will immeidately revoke access of old password. 
while 99999 means old password will still valid for 99999 minutes. but no admin will gonna do that in production environment.